dll/win32/shimgvw/resource.h

```diff
@@ -28,6 +28,8 @@
 #define IDB_MODIFY      64
 #define IDB_HELP_TOC    65
 
+#define IDC_TOOLBAR     42
```

Comment:
```suggestion
#define IDC_TOOLBAR     40
```
Please start groups at multiples of 10

Reply:
Why? This just needs to be outside the range of the standard IDOK, IDCANCEL etc.